scripts/build.sh

```diff
@@ -38,7 +38,7 @@ cp LICENSE "${TEMP_DIR}/"
 ls -al "${TEMP_DIR}"
 
 if [[ "${TARGET_PLATFORM}" != win-* ]]; then
-    MICROMAMBA_VERSION=1.5.11
+    MICROMAMBA_VERSION=2.0.6
```

Comment:
Were the SSL issues fixed in this one? I think so, but can't recall right now. Even if it they were not, I don't think they are an issue in the installers because we are not fetching anything (`git`-like or otherwise).

Reply:
https://github.com/mamba-org/mamba/pull/3765 is part of 2.0.6, but other SSL issues on Windows (also observed on others package managers) might still be there.